charts/seed-bootstrap/charts/monitoring/templates/config.yaml

```diff
@@ -68,6 +68,30 @@ data:
         - prometheus-web.garden.svc
       metric_relabel_configs:
 {{ include "prometheus.keep-metrics.metric-relabel-config" .Values.allowedMetrics.cadvisor | indent 6 }}
+
+    - job_name: kube-state-metrics
+      # Service is used, because we only care about metric from one kube-state-metrics instance
+      # and not multiple in HA setup
+      kubernetes_sd_configs:
+      - role: service
+        namespaces:
+          names: [ garden ]
+      relabel_configs:
+      - source_labels: [ __meta_kubernetes_service_label_component ]
+        action: keep
+        regex: kube-state-metrics
+      - source_labels: [ __meta_kubernetes_service_port_name ]
+        action: keep
+      - target_label: instance
+        replacement: kube-state-metrics
+      metric_relabel_configs:
+      - source_labels: [ pod ]
+        regex: ^.+\.tf-pod.+$
+        action: drop
```

Comment:
I guess, there shouldn't be any `tf-pod` metrics in the non-shoot namespaces, right?

Reply:
I wasn't entirely sure so I kept the relabel configs, but if we don't have any tf-pods in non-shoot namespaces I will remove this so we don't have unnecessary config.